src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -8252,6 +8252,37 @@ public void ParamsArray_Symbol_MultipleParamsArrays()
             Assert.True(((SourceParameterSymbol)lambdas[1].Parameters[2]).IsParams);
         }
 
+        [Fact]
+        public void ParamsArray_Symbol_ExternalReference()
+        {
+            var source = """
+                static void Report(object obj) => System.Console.WriteLine(obj.GetType());
+                var lam1 = (params int[] xs) => xs.Length;
+                Report(lam1);
+                var lam2 = (int[] xs) => xs.Length;
+                Report(lam2);
+                var lam3 = (int[] xs, params int[] ys) => xs.Length + ys.Length;
+                Report(lam3);
+                """;
+            var comp = CreateCompilation(source).VerifyDiagnostics();
+            CompileAndVerify(comp, expectedOutput: """
+                <>f__AnonymousDelegate0
+                System.Func`2[System.Int32[],System.Int32]
+                <>f__AnonymousDelegate1
+                """);
+            var reference = comp.EmitToImageReference();
+            var assembly = CreateCompilation("", new[] { reference }).GetReferencedAssemblySymbol(reference);
```

Comment:
We usually use a different, more efficient approach to verify symbols produced for emitted metadata on import. ```CompileAndVerify``` helper has a delegate parameter called ```symbolValidator```. If you get symbols from the module passed to that delegate as a parameter, you get symbols equivalent to those you are getting below.

Reply:
Thanks, changed.